javascript/big-practice/src/scripts/views/listNoteView.js

```diff
@@ -82,6 +85,34 @@ export default class ListNoteView {
     }
   }
 
+  /**
+   * @description function show empty note if that note is empty
+   *
+   * @param {Array} list is a list of note or list of trash note
+   * @param {String} type is a type if we need to use in listNotes or trashNotes
+   */
+  showHideEmpty(list, type) {
+    if (type === 'listNotes') {
+      if (!list.length) {
+        this.listNotesEmpty.classList.remove('hide');
+        this.listNoteElement.classList.add('hide');
+      } else {
+        this.listNotesEmpty.classList.add('hide');
```

Comment:
Why do we have a different naming between `listNotesEmpty` and `listNoteElement`?